source/val/validate_extensions.cpp

```diff
@@ -1082,6 +1082,23 @@ spv_result_t ValidateExtInstImport(ValidationState_t& _,
   return SPV_SUCCESS;
 }
 
+spv_result_t ValidateExtInstWithForwardRefs(ValidationState_t& _,
```

Comment:
I see this seems checked in validate_id.cpp. So maybe just the common validation code is necessary here (i.e. refactor OpExtInst checks).

Reply:
Function removed